spec/js/numbers/rbnf.spec.js

```diff
@@ -53,6 +52,9 @@ for (var locale in TwitterCldr.RBNF.test_resource) {
       if (formatter.rule_set_names_for_group(rule_group).indexOf(rule_set) === -1)
         continue;
       for (var test_case in TwitterCldr.RBNF.test_resource[locale][rule_group][rule_set]) {
+        if (!process.env.FULL_SPEC && Math.random() > 0.1) {
```

Comment:
Hmm... I'm not a big fan of running random tests. What about only running a certain, deterministic set of the tests? The Ruby version [only runs the spellout-numbering tests](https://github.com/twitter/twitter-cldr-rb/blob/master/spec/formatters/numbers/rbnf/rbnf_spec.rb#L57) during a fast test run. What do you think?